pkg/osutil/osutil.go

```diff
@@ -89,8 +91,8 @@ func CopyDir(
 			continue
 		}
 
-		srcPath := src + "/" + f.Name()
-		destPath := dest + "/" + f.Name()
+		srcPath := filepath.Join(src, f.Name())
```

Comment:
i'm pretty sure this is what's tanking the windows integration tests, since the embedded filesystems use the linux path separator even on windows
https://github.com/golang/go/issues/45230

Reply:
looks like this could break our FS abstraction since we need to know whether the fs is embedded to determine the separator

Reply:
yup just got it to work by switching these lines to from `filepath.join` (which uses the os separator to `path.join` whick exclusively uses the unix path separator.
Maybe we leave them reverted since that's how it already was and make a separate PR for sorting out the FS/embed os separator issues?